publisher-sdk/src/main/java/com/criteo/publisher/context/UserData.java

```diff
@@ -29,7 +29,7 @@
    * The hashing should be the users’ email address:
    * <ul>
    *   <li>Encoded in UTF-8</li>
-   *   <li>Trimmed of any white space (eg: “test@criteo.com “ should become “test@criteo.com”)</li>
+   *   <li>Trimmed of any white space (eg: “test@criteo.com “ should become “test@criteo.com“)</li>
```

Comment:
Do you want this to be part of that pull request ?

Reply:
When pushing to maven local, the javadoc task was complaining about illegal character. I don't know why it is not working locally. But doing a PR dedicated for is kind of overkill. Let me put this in a dedicated commit if it's ok for you.

Reply:
Thanks for the explaination !